layouts/partials/footer.html

```diff
@@ -42,19 +42,11 @@
     <script src="{{ path.Join "_" (index .Site.Data "rev-manifest" "js/site.js") | relURL }}"></script>
     <script src="{{ path.Join "_" (index .Site.Data "rev-manifest" "js/vendor/highlight.js") | relURL }}"></script>
     <script src="{{ path.Join "_" (index .Site.Data "rev-manifest" "js/vendor/svg4everybody.js") | relURL }}"></script>
-    <script type='application/ld+json'>
-        {
-          "@context": "http://schema.org",
-          "@type": "Organization",
-          "name": "Apache Camel",
-          "url": "{{ .Site.BaseURL }}",
-          "sameAs": [
-             "https://twitter.com/cameltweet"
-          ],
-          "logo": "{{ .Site.BaseURL }}_/img/logo-d.svg",
-          "description": "Apache Camel ™ is a versatile open-source integration framework based on known Enterprise Integration Patterns. Camel empowers you to define routing and mediation rules in a variety of domain-specific languages, including a Java-based Fluent API, Spring or Blueprint XML Configuration files, and a Scala DSL."
-        }
-    </script>
-    {{ partial "breadcrumbs.html" . }}
+    <script src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/selectize@0.12.6/dist/js/standalone/selectize.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/algoliasearch@3.30.0/dist/algoliasearchLite.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@beta"></script>
```

Comment:
I'd prefer if we vendor all dependencies. Not sure how much more do these dependencies add to the page weight. In general I would try to minimize the 3rd party dependencies in the website runtime. JQuery is 85KB and in total we're currently serving 28KB, so that would be almost 3x increase in JavaScript. I know both numbers aren't that big, but currently we have sub second time to first meaningful paint and I'm afraid of putting additional weight on the page might jeopardize that.